PoseEstimation-TFLiteSwift/StillImageLineViewController.swift

```diff
@@ -198,6 +198,17 @@ class StillImageLineViewController: UIViewController {
         navigationController?.present(pickerVC, animated: true)
     }
     
+    @IBAction func export(_ sender: Any) {
+        guard let overlayViewRect = overlayLineDotView?.frame else { return }
+        let image = view.uiImage(in: overlayViewRect)
+        let imageData = image.jpegData(compressionQuality: 0.95)
+        let paths = FileManager.default.urls(for: .documentDirectory, in: .userDomainMask)
+        let fileURL = paths[0].appendingPathComponent("pose-linedot-demo.jpeg")
+        try? imageData?.write(to: fileURL)
+        let vc = UIActivityViewController(activityItems: [fileURL], applicationActivities: nil)
+        present(vc, animated: true)
```

Comment:
I think, export has 2 tasks.

 1. export Image.
 2. show UIActivityViewController.

I recommend to make 2 abstract task(exportImage, showUIActivityViewController).

Reply:
@syjdev 
Thanks for your opinion. 
I think the above function has exporting behavior in the app term. So I'll separate the `export(_:)` function to saving the image step and show `UIActivityViewController` step. If you have any other comments, reply feel free.

Thank you.